src/test/ui/issues/issue-31299.rs

```diff
@@ -30,5 +30,5 @@ struct PtrBack<T: Front>(Vec<T::Back>);
 struct M(PtrBack<Vec<M>>);
 
 fn main() {
-    std::mem::size_of::<M>();
+    let _ = std::mem::size_of::<M>();
```

Comment:
For this one, I think you should ignore the lint rather than adding `let _ =`, just in case it affects the test.

Reply:
You got it.